components/infra/Makefile

```
@@ -256,49 +256,37 @@ endef
-		if vault secrets list -format=json | grep -q "\"$(TRANSIT_MOUNT)/\""; then \
-			echo "transit already enabled at $(TRANSIT_MOUNT)/"; \
+		if vault secrets list -format=json | grep -q "\"$(TRANSIT_MOUNT_ST)/\""; then \
+			echo "transit already enabled at $(TRANSIT_MOUNT_ST)/"; \
 		else \
-			vault secrets enable -path=$(TRANSIT_MOUNT) transit && echo "enabled transit at $(TRANSIT_MOUNT)/"; \
+			vault secrets enable -path=$(TRANSIT_MOUNT_ST) transit && echo "enabled transit at $(TRANSIT_MOUNT_ST)/"; \
 		fi'
 	@echo "$(GREEN)$(BOLD)[ok]$(NC) Single-tenant transit ready$(GREEN) ✔️$(NC)"
 
 .PHONY: vault-transit-multi
 vault-transit-multi:
```

checkmake (0.3.2)

[warning] Target body for "vault-transit-multi" exceeds allowed length of 5 lines (9).

(maxbodylength)